docs/state.md

```diff
@@ -40,25 +40,28 @@ The worker maintains a worker state which is a JSON representation of the follow
 
 ```jsonc
 {
-    "worker_id": "<WORKER_ID>"
+    "worker_id": "<WORKER_ID>",
+    "instance-id": "<INSTANCE_ID>"  # OPTIONAL
```

Comment:
Looking at the code, I think this should be `instance_id`, right? Same with the docs below